### PR TITLE
node-16-remove-node-sass

### PR DESCRIPTION
**Pull request by sikhote**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,14 +46,14 @@
     "eslint-plugin-vue": "^4.3.0",
     "jest": "^22.4.3",
     "jest-serializer-vue": "^1.0.0",
-    "node-sass": "^5.0.0",
-    "poi": "^9.6.13",
+    "poi": "^12.10.3",
```
Comment (aamathews23) [inline package.json:49]: I wonder once we migrate to Vue 3 if we can get rid of `poi` and `rollup-plugin-vue` in favor of `vite` 🤔

File: package.json [hunk 2]
```diff
@@ -9,14 +9,18 @@
     "url": "MikaelEdebro/vue-airbnb-style-datepicker",
     "type": "git"
   },
+  "engines": {
```
Comment (aamathews23) [inline package.json:12]: 👍

**Review by aamathews23 — APPROVED** [review]
Looks good. Nice easy upgrade 😄

File: package.json (hunk 1 above)
Comment (sikhote) [reply package.json:49]: Yes for sure 🙏  I think I heard they are completely taking this internal once we do Vue 3. I'm kinda expecting this to be a temporary solution for now.

**Pull request merged**